azext_iot/product/shared.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from enum import Enum
+
+
+class TaskType(Enum):
+    QueueTestRun = "QueueTestRun"
+    GenerateTestCases = "GenerateTestCases"
+
+
+class BadgeType(Enum):
+    IotDevice = "IotDevice"
+    Pnp = "Pnp"
```

Comment:
Does the service expect "Pnp"? The short reference we've used elsewhere is "PnP" lol

Reply:
Yep: `Available values : IotDevice, Pnp, IotEdgeCompatible`